README.md

```diff
@@ -1,6 +1,7 @@
 # boonafide
+<!---
 [![Codacy Badge](https://api.codacy.com/project/badge/Grade/032683a5d73b414688fbf6fb6967d909)](https://www.codacy.com/app/aemartos/boonafide?utm_source=github.com&amp;utm_medium=referral&amp;utm_content=aemartos/boonafide&amp;utm_campaign=Badge_Grade)
-
+-->
 ### [https://boonafide.herokuapp.com/](https://boonafide.herokuapp.com/)
```

Comment:
Just commented this because it seems broken, feel free to undo this change

Reply:
I've fixed it TY!